.github/workflows/publish_packages.yaml

```
@@ -0,0 +1,27 @@
+      - "*-v[0-9]+.[0-9]+.[0-9]+*"
+
+jobs:
+  publish:
+    if: github.ref_type == 'tag'
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+    steps:
+      - name: Extract package name from the tag
+        id: extract
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:3:38: Double quote to prevent globbing and word splitting

(shellcheck)